package.json

```diff
@@ -4,10 +4,12 @@
   "main": "server.js",
   "license": "MIT",
   "dependencies": {
+    "@microsoft/microsoft-graph-client": "^2.0.0",
     "@ideditor/country-coder": "^3.2.0",
     "buffer": "^5.2.1",
     "crypto-js": "^3.1.9-1",
     "d3-geo": "^1.11.6",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
how come this was needed?

Reply:
It's a dependency of the Microsoft graph JavaScript SDK. They need a polyfill of fetch to make sure everything from fetch is available. It might not be needed in the current context though. See https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/dev/README.md#via-npm

Reply:
Hmm it would be best not to add it as it might conflict with the way we use `fetch` in react-native which is not using isomorphic-fetch.

Reply:
If react native implements the whole fetch specification and is promise compliant I don't see why we'd need it.

Reply:
We use native fetch already internally, so this should work out of the box. Let's remove it!

Reply:
removed